README.md

```diff
@@ -5,19 +5,22 @@
 Use the package manager [pip](https://pip.pypa.io/en/stable/) to install requirements.
 
 ```bash
+python3 -m venv venv
+source venv/bin/activate
 pip install -r requirements.txt
 ```
 
 ## Usage
 
-Change the config.py file by entering in your email adresses and a gmail generated API key.
+Copy the `configuration.example.json` file into `configuration.json` and put in your email adresses and a gmail generated API key.
```

Comment:
could you rename this file?
`configuration.example.json` => `configuration_example.json`

Reply:
done, you can merge :)